src/test/fuzz/fuzz.cpp

```diff
@@ -245,6 +245,10 @@ int main(int argc, char** argv)
         test_one_input(buffer);
         return 0;
     }
+#ifdef WIN32
```

Comment:
```suggestion
#ifdef _WIN32
```

`_WIN32` is a compiler flag and could be used for both windows platform i.e windows_x86 (32-bit) as well as windows_x64 (64-bit)

Reply:
The current code is consistent with our entire code base.